AndroidSDKCore/src/main/java/com/leanplum/internal/RequestOld.java

```diff
@@ -454,7 +454,7 @@ protected RequestsWithEncoding getRequestsWithEncodedStringStoredRequests(double
         requestsToSend = new ArrayList<>(0);
       } else {
         unsentRequests = getUnsentRequests(fraction);
-        requestsToSend = removeIrrelevantBackgroundStartRequests(unsentRequests);
+        requestsToSend = new ArrayList<>(unsentRequests);
```

Comment:
Does this ensure that start action will not be called more than once if it has failed? This might probably happen at the next start of the app only.

Reply:
The start action is sent by sendIfConnected so every time the app is opened all batched events are sent. If I stop the data on the device and try to simulate multiple start events by killing the app and running it again it produces the following sequence of events:

start, pauseSession, start, pauseSession, start, pauseSession and so on.

Cannot reproduce batch with multiple start events one after another so decided to fully remove this removeIrrelevantBackgroundStartRequests method.